content/2016.06.ab-testing-with-kinto.rst

```diff
@@ -0,0 +1,199 @@
+How to use Kinto to A/B test a web application?
+###############################################
+
+:lang: en
+:date: 2016-06-05
+:url: ab-testing-with-kinto
+:summary: A/B testing is a form of statistical hypothesis testing. Its
+          goal is to identify changes to web pages that increase or
+          maximize an outcome of interest. Let's see how we can use
+          kinto to use this technic on our website.
+
+
+Introduction
+============
+
+I am not a big fan of A/B testing because I think, most of the time,
+`we can do better`_ however in `certain circumstances`_ A/B testing is
+still a really good idea to test a feature.
+
+.. image:: {filename}/images/kinto-logo.png
+    :alt: The Kinto Logo
+
+
+In an other hand, I am a big fan of the philosophy behind Kinto_ which
+as you may know is a database for the web.
+
+In this article, I will take you through a little experiment I did
+lately to extend the `work that have been done`_ to activate a feature
+for some users in Firefox for Android.
+
+.. _`we can do better`: http://stevehanov.ca/blog/index.php?id=132
+.. _`certain circumstances`: https://vwo.com/blog/multi-armed-bandit-algorithm/
+.. _`work that have been done`: https://github.com/mozilla-services/switchboard-experiments/
+
+
+How does A/B testing works?
+===========================
+
+You just created a really nice feature and you want to test multiple
+UX solution in order to decide which one is most usable for your users.
+
+For instance, what is the best wording for the french banner?
+
+You can use A/B testing to test differents variations on your user
+base and select the best option.
+
+There is two technical parts on this:
+
+1. You need to display different version of your button to your users,
+   probably with the same code.
+2. You need to be able to get the conversion rate for each solution to
+   be able to compare them.
+
```

Comment:
I don't find this example/explanation very explicit. 

Something like that maybe? `You want to know which color is the best for your sign-up button. You make two versions, each one is shown to a proportion of users, and you count which color receives the highest number of clics.`